### ISSUE TITLE
Cannot exclude QuickMiner from Bitdefender protection

### ISSUE THREAD

**Issue by laurian-lamba** [nicehash/NiceHashQuickMiner#213]
It seems impossible to install Quick Miner when running BitDefender as a security tool in Windows 10. No matter what exceptions are defined the excavator.exe application is not allowed.

**Comment by nicehashdev** [comment]
We have plans to launch global (all-AV) whitelist campaign shorty. Currently, we actively keep it not to be unjustly blocked by Microsoft or Google only.

I suggest you to take this matter to the BitDefender's customer support service and explain them the issue.

**Comment by laurian-lamba** [comment]
Okay, but a release process for a secure product is that you first take care of the whitelisting campaign with major security vendors, before rolling out a stable product. It shouldn't be the end user's responsibility.

So, then this is an "unsecure" product until it is whitelisted.

**Comment by nicehashdev** [comment]
It is **not** unsecure.  Code is signed by EV code certificates which are in the first place not so easy to obtain and secondly there is a real company behind vouching for safety of the application.

Not all companies have resources to fight all AV vendors - this is expensive. So far, we have been covering only Microsoft and Google and we are, as becoming larger, starting to fight others, too.

**Comment by laurian-lamba** [comment]
Then it would be nice to have these statements in a security policy inside your repos.
